setup.py

```diff
@@ -52,6 +52,8 @@ def rel(*xs):
     "pytest-benchmark[histogram]",
     "pytest-cov",
     "tox",
+    "freezegun",
+    "python-dateutil>=2.8.0",
```

Comment:
you put this into dev dependencies, it should be next to pytz. As it's not only for the tests